Source/Core/VideoCommon/VertexShaderManager.cpp

```diff
@@ -407,7 +414,9 @@ void VertexShaderManager::SetConstants()
 		bProjectionChanged = false;
 
 		float *rawProjection = xfmem.projection.rawProjection;
-
+		float temp_ratio;
+		float vp_ratio;
+		bool viewport_matches;
```

Comment:
These should be declared where the assignments are, since they're only used in one case (just brace said case).